Develop

OlgaMarsal opened the pull request.

OlgaMarsal committed:
LinerLayout calculator lesson 212

OlgaMarsal committed:
TableLayout calculator lesson 212

OlgaMarsal committed:
ConstraitLayout calculator lesson 212 - part 2

OlgaMarsal committed:
ConstraitLayout calculator lesson 212 - part 3 background

OlgaMarsal committed:
ConstraitLayout calculator lesson 212 - lesson 3

Sandor13 commented on app/src/main/java/ru/geekbrains/lesson212/MainActivity.java:
```diff
@@ -3,12 +3,181 @@
+    private String number;
+    private String point = ".";
+    private TextView numbersText;
+    private Operations operations;
```
можно инициализировать прямо тут

Sandor13 commented on app/src/main/java/ru/geekbrains/lesson212/MainActivity.java:
```diff
@@ -3,12 +3,181 @@
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private String number;
+    private String point = ".";
```
если point никогда не меняется , то нужно сделать его final static

Sandor13 commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -2,22 +2,26 @@
-        android:src="@drawable/background" />
-
+        android:layout_height="50dp"
+        android:layout_marginBottom="20dp"
```
в ресурсы

OlgaMarsal committed:
lesson 214 - HW4 внесла исправления комментариев к дз 3 урока, но

android:layout_height="wrap_content" не получается убрать в ресурсы
алт ентер нажимаю высвечивается только оверайт...
android:background="@drawable/background"> то же самое

OlgaMarsal committed:
lesson 214 - HW4 материал

OlgaMarsal committed:
lesson 214 - HW4 стайл. Осталось понять как прописать оперейшенс. мне кажется что еще логику калькулятора можно было бы сделать как объяснили на уроке через каунтер, где все начинается с 0, как в калькуляторе, а дальше каунтер через математическую логику

OlgaMarsal committed:
lesson 215 - HW5 динамика тем
